src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -24,26 +24,101 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-set -ex
+USAGE="
+usage: install_dependencies_and_build.sh [options]
+
+Sets up python execution environment for AWS Neuron SDK for execution on Inferentia chips.
```

Comment:
Is this copied from somewhere or for a different ticket? This comment seems unrelated.

Reply:
good catch. updated!